src/tests/topp/THIRDPARTY/CometAdapter_6_out.idXML

```diff
@@ -88,10 +88,10 @@
 				<UserParam type="string" name="MS:1002259" value="20"/>
 				<UserParam type="string" name="num_matched_peptides" value="2"/>
 				<UserParam type="int" name="isotope_error" value="0"/>
-				<UserParam type="float" name="MS:1002252" value="0.1106"/>
-				<UserParam type="float" name="COMET:xcorr" value="0.1106"/>
-				<UserParam type="float" name="MS:1002253" value="1.0"/>
-				<UserParam type="float" name="COMET:deltaCn" value="1.0"/>
+				<UserParam type="float" name="MS:1002252" value="0.111"/>
+				<UserParam type="float" name="COMET:xcorr" value="0.111"/>
+				<UserParam type="float" name="MS:1002253" value="0.0"/>
+				<UserParam type="float" name="COMET:deltaCn" value="0.0"/>
```

Comment:
maybe quick check why this is zero now

Reply:
Changes to how Comet handles delta Cn for single hits:
"Change the convention for the dCn (delta Cn) score for single hit results i.e. those results where only a single peptide is scored/reported. In the past, these single peptide hits received a dCn score of 1.0 but now these single peptide hits will receive a dCn score of 0.0." see https://github.com/UWPR/Comet/releases/tag/v2024.01.0